package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "chrome-extensions",
+  "version": "1.0.0",
```

Comment:
How does this one relate to manifest version?

Reply:
It does not...
The package.json is only there to specify the eslint dependencies., and the command-lines that can be executed with `npm run eslint`

"name" and "version" are required fields in jackage.json, but their contents are not actually used.